docs/data/versions.json

```diff
@@ -2,7 +2,8 @@
 	"params": {
 		"currentVersion": "v1",
 		"ltsVersion": "v1",
-		"previousVersions": []
+		"previousVersions": [],
+		"downloadVersions": ["v1"]
```

Comment:
What are the semantics of this? Is the idea that these are "extra" versions to download in addition to "previousVersions"? Will this go away after we light up v2 docs? Just want to make sure I understand.

Reply:
I was intending to keep the new field as "downloadVersions" since it may better represent the versions that needs to be manually uploaded/downloaded. Previous versions will probably track the previous major versions which can be extracted from the pipelines.